.vscode/settings.json

```diff
@@ -0,0 +1,35 @@
+{
```

Comment:
Diese Datei bitte aus dem repo löschen und ins .gitignore aufnehmen (lokale editor-settings).

Reply:
I deleted the file